### ISSUE TITLE
clashes with auto-suggestion

### ISSUE THREAD

**Issue by mimoo** [marlonrichert/zsh-autocomplete#57]
Hello!

Let's say I want to do `git add something`, the way I would usually do this is to write `git add som` and use the autocomplete. But the `tab` key is now broken and I see:

```
No matching completions found.
```

Is there a solution to this?

**Comment by marlonrichert** [comment]
@mimoo @jef I cannot reproduce this with my local build. In your clone of the `zsh-autocomplete` repo, can you please `git pull; git checkout dev`, restart your terminal and try if it works then?

**Comment by marlonrichert** [comment]
Here's what I did to reproduce it:
```shell
% cd "$(mktemp -d)"
ZDOTDIR=$PWD HOME=$PWD zsh -df
git init
touch foo
git add f
```
Result:
* The completion menu appears and shows `foo`.
* Pressing <kbd>Tab</kbd> results in `f` being completed to `foo`.

**Comment by jef** [comment]
Let me try again today. I may have had a fluke.

**Comment by jef** [comment]
Worked fine for me. I think my problem was with aliasing `git` with something like `dotfiles`

```
dotfiles='/usr/bin/git --git-dir=/Users/anon/.dotfiles/ --work-tree=/Users/anon'
```

Then running something like:

```
dofiles add f
```

The autocompletion works for `foo`, but not necessarily for `add` or other git options. Could be more of a complex thing going on here though.

**Comment by marlonrichert** [comment]
OK, good to know it's not a problem with `zsh-autocomplete`, then. 🙂

**Comment by jef** [comment]
I wouldn't be so sure that it's not a `zsh-autocomplete` problem as it should be completing aliases as well.

For example:

```
alias g=git
g add f
```
